journal/modernized/13-chapter-12.adoc

```diff
@@ -1,11 +1,13 @@
-== Section XII.
+== Chapter XII
 
-Of his frequently exposing his own life to save his neighbours,
-and standing his ground in those perilous times, till his house was set on fire,
-himself and two sons led away to be murdered by the Irish raparees;
-yet by Divine Providence were delivered out of their hands.
-About the year 1690, he, with many more, saw the fulfilling of his prophecy,
-mentioned in the last section.
+[.chapter-synopsis]
+* His frequently exposing his own life to save his neighbours,
+* and standing his ground in those perilous times, till his house was set on fire,
+* himself and two sons led away to be murdered by the Irish raparees;
+* yet by Divine Providence were delivered out of their hands
+* About the year 1690, he, with many more, saw the fulfilling of his prophecy,
+* mentioned in the last section
+* //Not sure what to do with this one!!
```

Comment:
Ok, so this is the one that we were slacking about. I would maybe arrange this one like this:
```ASCIIDOC
[.chapter-synopsis]
* His frequently exposing his own life to save his neighbours, and standing his ground in those perilous times
* His house set on fire; himself and two sons led away to be murdered by the Irish Raparees
* Delivered out of their hands by Divine Providence
* About the year 1690, he, with many more, see the fulfilling of his prophecy, mentioned in the last section
```
I don't know if Jared told you, but you don't have to worry about capitalizing the key words of these synopsis sections any more. The linter will do it automatically.👍🏻

Reply:
Yes, he let me know 😄